protocol/x/clob/flags/flags.go

```diff
@@ -33,8 +34,9 @@ const (
 
 // Default values.
 const (
-	DefaultMaxLiquidationOrdersPerBlock    = math.MaxUint32
-	DefaultMaxDeleveragingAttemptsPerBlock = 35
+	DefaultMaxLiquidationOrdersPerBlock        = 20
+	DefaultMaxDeleveragingAttemptsPerBlock     = 5
+	DefaultMaxDeleveragingSubaccountsToIterate = 500
```

Comment:
@BrendanChou are these numbers reasonable? 

I think after the metrics optimization, we might be able to increase these numbers (maybe 3x or 5x, given the performance improvement). let me know what you think

Reply:
I'd personally prefer the more aggressive numbers Brendan suggested unless the performance tradeoff is meaningful (liquidating / deleveraging bad debt quickly is important for the exchange).